### ISSUE TITLE
hook select crashes 

### ISSUE THREAD

**Issue by Hanspe-3019** [ttscoff/hookapp#7]
Running on Intel Core i5:
```
% hook select https://www.nachdenkseiten.de       
sh: line 2: /Library/Ruby/Gems/2.6.0/gems/hookapp-2.0.8/lib/hook/../helpers/fuzzyfilefinder: Bad CPU type in executable
```

```
hookapp-2.0.8/lib/helpers % file fuzzyfilefinder 
fuzzyfilefinder: Mach-O 64-bit executable arm64

```
Can to provide a universal file?

**Comment by ttscoff** [comment]
Ah, I ran into this issue with my doing project as well. The solution I came up with was to build the fzf binary at runtime, so I'll need to work that functionality into the hook CLI. In the meantime, if you run `brew install fzf` and copy the fzf binary to `/Library/Ruby/Gems/2.6.0/gems/hookapp-2.0.8/lib/helpers/fuzzyfilefinder` (renaming to fuzzyfilefinder) it should allow the command to run for you.

**Comment by Hanspe-3019** [comment]
Did install 2.0.9. Got error when executing hook:
```
/usr/local/bin/hook:23:in `load': cannot load such file -- /Library/Ruby/Gems/2.6.0/gems/hookapp-2.0.9/bin/hook (LoadError)
	from /usr/local/bin/hook:23:in `<main>'
```
Cause: File mode in hookapp-2.0.9/** not set for group and other.
Run `sudo chmod +r ...`

Now hook-cli seems to work. Because fzf is already in my path, I did not test installation helper for fzf. I inspected the files in hookapp-2.0.9: The directory `helpers` is missing, so the install should not succeed anyway.

**Comment by ttscoff** [comment]
I think I've fixed the permissions issues in 2.0.10. I ran into a 
similar problem with another one of my gems, I think the switch to a new 
Mac changed some local permissions without me realizing it.

The helpers directory would be at 
$GEM_PATH/gems/hookapp-2.0.10/lib/helpers, but is only be created if it 
actually needs to install fzf. If it finds it locally, it should just 
use your native copy. Does `hook select FILENAME` work for you now, or 
still crashing?

-Brett
